pom.xml

```diff
@@ -144,6 +158,7 @@
                             <goal>deploy</goal>
                         </goals>
                         <configuration>
+                            <jboss-home>${jboss.home}</jboss-home>
```

Comment:
This is necessary for the correct version of the provisioned server to start:

```bash
➤ mvn clean install '-Dversion.org.wildfly=33.0.0.Beta1-SNAPSHOT' '-Dversion.wildfly-maven-plugin=5.0.0.Final' '-Pprovision-preview'
...
20:21:15,549 INFO  [org.jboss.as] (MSC service thread 1-2) WFLYSRV0050: WildFly Preview 33.0.0.Beta1-SNAPSHOT (WildFly Core 24.0.0.Final) stopped in 55ms
```

Reply:
The build log: https://github.com/liweinan/wildfly-preview-ci-experiment/actions/runs/9032148412/job/24820091391?pr=1#step:6:42785